apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestAudioActivity.java

```diff
@@ -820,43 +941,93 @@ protected void toastPauseError(int result) {
     }
 
     public void pauseAudio() {
-        int result = pauseNative();
+        synchronized (mAudioStateLock) {
+            pauseAudioLocked();
+        }
+    }
+
+    @GuardedBy("mAudioStateLock")
+    private void pauseAudioLocked() {
+        int result;
+        if (mAudioState != AUDIO_STATE_STARTED) {
```

Comment:
Please do not silent the pause call. OboeTester should be able to call pause and let the lower layer decide if it should success or not.